doc/install.rst

```diff
@@ -32,6 +35,16 @@ Installation From Source
     ./configure --prefix "$(dirname $(pwd))" && make && make install
     popd
 
+If you want to connect to `SSL` enabled databases you need to configure `FreeTDS` with the `--with-openssl` option. OpenSSL is
+generally found in `/usr/lib/ssl`, so the configuration command will look like this.
+
+.. code-block:: bash
+   ./configure --prefix "$(dirname $(pwd))" --with-openssl='/usr/lib/ssl'
+   
+FreeTDS also have a configure time option (`--with-tdsver`) to set the default TDS version being used. This defaults to 5.0 if not provided. So it might be a good idea to set it to a modern version like 7.3. Now the configuration command would look like this.
```

Comment:
Note that ctds will default to the latest supported TDS version. If you were seeing issues connecting to Azure when not specifying a `tds_version`, it was likely because ctds was defaulting the TDS version to *7.4*.

Reply:
Ok, I got the default version from FreeTDS docs. I'll correct this.